### PR TITLE
Use the modular player

### PR DESCRIPTION
**Pull request by bitmovin-kenny**
When testing on the Tizen 2016 i came across some performance issues and one step toward improvement was to only load the minimally required modules.

This PR updates the demo accordingly

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -41,20 +53,25 @@ function setupPlayer() {
...
 
 	var container = document.getElementById('player');
-	var player = new bitmovin.player.Player(container, conf);
+	// expose on window object for the key controller event handlers
+	window.player = new bitmovin.player.core.Player(container, conf);
 	
 	player.load(source).then(function(value) {
 		// Success
 		console.log("Successfully created bitmovin player instance");
+		player.buffer.setTargetLevel(bitmovin.player.core.BufferType.BackwardDuration, 10, bitmovin.player.core.MediaType.Video);
```
Comment (protyposis) [inline js/main.js]: You can already set the buffer levels before `load`. But why don't you do it directly in the config?

Comment (bitmovin-kenny) [reply js/main.js]: I actually forgot about the buffer config and thought this was the only way to set the buffer levels now, will update

File: index.html [hunk 2]
```diff
@@ -14,8 +14,18 @@
...
-    <!-- Bitmovin player -->
-    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
+    <!-- Bitmovin player modules -->
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-core.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-polyfill.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-engine-bitmovin.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-mserenderer.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-abr.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-drm.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-container-mp4.js"></script>
```
Comment (zachmanc) [inline index.html:24]: can we add `container-ts` as well? I think a decent amount of our users might be using HLS + ts as well

File: index.html [hunk 3]
```diff
@@ -14,8 +14,18 @@
...
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-xml.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-dash.js"></script>
```
Comment (zachmanc) [inline index.html:27]: same comment here, can we add hls as well?

File: index.html (hunk 2 above)
Comment (bitmovin-kenny) [reply index.html:24]: Sure, i just thought i would take the minimal amount of modules for the one source which is loaded. 

What about ads and subtitles? 😄

Comment (zachmanc) [reply index.html:24]: Added HLS. left off ads and subtitles

File: js/main.js (hunk 1 above)
Comment (zachmanc) [reply js/main.js]: updated to use the buffer config

**Review by jonathanhperry — APPROVED** [review]
I was able to get DASH, HLS and DASH+Widevine test streams to work with this config setup on a physical Samsung Tizen TV

**Pull request merged**